packages/ui/src/components/ui/legacy/calendar/event-form-components.tsx

```
@@ -114,25 +114,219 @@
+      if (textareaRef.current) {
+        // Scroll the button into view when toggling
+        const button = textareaRef.current.parentElement?.parentElement?.querySelector('button[data-show-toggle]');
+        if (button) {
+          button.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
+        }
+      }
+    }, 0);
+  };
+
+  // Throttle function
+  const throttle = (func: Function, limit: number) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)